cabal.release.project

```diff
@@ -7,7 +7,26 @@ index-state: hackage.haskell.org 2025-06-27T20:43:14Z
 
 -- never include this or its TH dependency in a release!
 package cabal-install
-  flags: -git-rev
+  flags: -git-rev -lukko
```

Comment:
1. Didn't we previously nuke the `lukko` stuff, or at least mark it as default off?
2. You should probably update the comment to indicate that it only applies to `-git-rev`.

Reply:
It is not nuked, the flag is just off by default. I want to enforce that release is always built without it, due to bugs on 32bit. If someone messes with the cabal file flags, then the release may break. This makes sure that can't happen. The releases in general should never depend on default flags. All flags needed should be explicit.